gef-ui/server/controllers/application-details/index.test.js

```diff
@@ -255,6 +257,59 @@ describe('controllers/application-details', () => {
           }));
       });
 
+      it('renders `application-details` with displayChangeSupportingInfo as true when draft', async () => {
+        api.getFacilities.mockResolvedValue(MOCKS.MockFacilityResponseChangedIssued);
+        mockApplicationResponse.status = CONSTANTS.DEAL_STATUS.DRAFT;
+        api.getApplication.mockResolvedValueOnce(mockApplicationResponse);
+
+        await applicationDetails(mockRequest, mockResponse);
+
+        expect(mockResponse.render)
+          .toHaveBeenCalledWith('partials/application-details.njk', expect.objectContaining({
+            displayChangeSupportingInfo: true,
+          }));
+      });
+
+      it('renders `application-details` with displayChangeSupportingInfo as false when preview mode (UKEF_IN_PROGRESS)', async () => {
+        api.getFacilities.mockResolvedValue(MOCKS.MockFacilityResponseNotChangedIssued);
+        mockApplicationResponse.status = CONSTANTS.DEAL_STATUS.UKEF_IN_PROGRESS;
+        api.getApplication.mockResolvedValueOnce(mockApplicationResponse);
+
+        await applicationDetails(mockRequest, mockResponse);
+
+        expect(mockResponse.render)
+          .toHaveBeenCalledWith('partials/application-details.njk', expect.objectContaining({
+            displayChangeSupportingInfo: false,
+          }));
+      });
+
+      it('renders `application-details` with displayChangeSupportingInfo as true submission count is 0', async () => {
```

Comment:
**when** `submission count is 0`

very minor sorry 😝

Reply:
Done